django_prices_vatlayer/utils.py

```diff
@@ -0,0 +1,50 @@
+from decimal import Decimal
+import requests
+
+from django.conf import settings
+from django.core.exceptions import ImproperlyConfigured, ObjectDoesNotExist
+from django.utils import six
+
+from .models import EuropeanVatRate
+
+try:
+    ACCESS_KEY = settings.VATLAYER_ACCESS_KEY
+except AttributeError:
+    raise ImproperlyConfigured('VATLAYER_ACCESS_KEY is required')
+
+RATINGS_URL = 'http://apilayer.net/api/rate_list'
+
+
+def validate_data(json_data):
+    if not json_data['success']:
+        info = json_data['error']['info']
+        raise ImproperlyConfigured(info)
+
+
+def get_european_vat_rates():
```

Comment:
I would prefer if the name reflected `download` or `fetch` as `get` kind of implies we hit the database and not an external API.